### ISSUE TITLE
Need a 'crash' sound effect

### ISSUE THREAD

**Issue by togfoxy** [togfoxy/Mars-Lander-reboot#24]

**Comment by DocMcSnappen** [comment]
https://soundimage.org/sfx-combat/

I personally like Explosion2 and Explosion8.  (This is Eric Matyas' website, in case you've never visited it before.)

**Comment by togfoxy** [comment]
Nice. Thanks! Do you think it should be
- on every contact with the ground
- every heavy contact
- Only when health is zero
- Something else?

**Comment by DocMcSnappen** [comment]
I think they all should have their own sounds, actually.  Or even a pool of sounds that can be randomly drawn from for variety.

One sound/pool for landing, one for light (non-landing) impact, one for heavy impact, and one for zero health.  There's probably other sounds that could be added, but this is a really good starting place.  I haven't done any sound stuff with Lua/Love yet, but it's a topic I've been wanting to learn.  :)

**Comment by togfoxy** [comment]
Good ideas!

There are already two sounds incorporated. Did you want to follow that example and suggest some code in your own time?

You'll need to note that the asset loader makes this a little bit different than normal.

**Comment by DocMcSnappen** [comment]
Yeah, I'll look into this when I have time - likely next week.  And thanks for the heads-up about the asset loader.  I'll make sure I understand it before monkeying.  :D
